packages/migration/src/util/MoveImports.ts

```diff
@@ -0,0 +1,35 @@
+import ts from 'typescript';
+import { Modder, TypescriptMod } from 'riceburn/lib/interfaces';
+
+export function generateImport(packageName: string, namedImports: string[]): string {
+  return `import { ${namedImports.join(', ')} } from '${packageName}'`;
+}
+
+export function moveImports(node: ts.Node, modder: Modder, fromPackage: string, toPackage: string, constantNames: string[]): TypescriptMod {
+  if (
+    ts.isImportDeclaration(node) &&
+    (node.moduleSpecifier.getText() === `'${fromPackage}'` || node.moduleSpecifier.getText() === `"${fromPackage}"`) &&
+    node.importClause &&
+    node.importClause.namedBindings &&
+    ts.isNamedImports(node.importClause.namedBindings)
+  ) {
+    const namedBindings = node.importClause.namedBindings;
+    const untouchedImports: string[] = [];
+    const migratedImports: string[] = [];
+    namedBindings.forEachChild(c => {
+      const importName = c.getText();
+      if (constantNames.indexOf(importName) >= 0) {
+        migratedImports.push(importName);
+      } else {
+        untouchedImports.push(importName);
+      }
+    });
+    if (migratedImports.length > 0) {
```

Comment:
Something about this logic is creating an extra weird `import { } from 'office-ui-fabric-react'` line in the snapshot

Reply:
I believe it's intentional, just had a few lines that were munged by prettier to look the same.